src/main.rs

```diff
@@ -52,13 +52,42 @@ use std::fs::File;
 use std::io::BufReader;
 use std::io::Read;
 use std::path::Path;
+use std::process::Command;
 
 use error::{Error, Result};
 
 static NOTFOUND: &[u8] = b"Not Found";
 
+// Runs checks required for Keylime to work on the system.
+fn check_system_prerequisites() -> Result<()> {
+    // Check Python version > 3.x.
+    // This is necessary for running revocation scripts.
+    let python_version = String::from_utf8(
+        Command::new("python")
```

Comment:
Note: One more thing here I need to double check -- but on CoreOS system python is the only provided python. I'll double check to see if it's in PATH or if it has to be found directly. Either way I don't see it as a blocker as if we need a second or expanded check we can add it later.

Reply:
Hmm, this seems important - I'd like to add it.

Reply:
Based on the discussion in our meeting earlier I have a feeling this won't be needed.